.travis.yml

```diff
@@ -1,17 +1,32 @@
+sudo: false
+
 language: node_js
+
+warnings_are_errors: false
+
 node_js:
-  - "node"
+  - 10
+
+branches:
+  only:
+    - master
+
 notifications:
-  email: false
-warnings_are_errors: false
-script:
-  - echo "TODO"
-  - npm run build
+  email:
+    on_failure: change
+    on_success: change
+
+cache:
+  yarn: true
+  directories:
+    - node_modules
+
+before_deploy:
+  - yarn build
+
 deploy:
-  provider: firebase
   skip_cleanup: true
-  on:
-    branch: master
+  provider: firebase
   token:
     secure: JR5wZs885tCKabT6mZtCkPi1XKHy6GJ+wymQCeho9zzkmnGs7zpjDBazs3+YS0nS3PBLSpwa8qEfMG6NctoZW25F7pJApJPjKZPkxe7NTLTMYHSesEjPXGBGFk2SrSGIvqWUfDCo9fxo+zEyKVeziRYLz+/UnaS7d2PBhu6JSX7aPVS0WdPpCFhQj1GZN+MFx0e3VdZCJBI8SsQNfM44bDQV4trpt1B0eQyMG7C+1muGA3MfTb3x26zClDK/4WisZFSxbi5xcaDgkJmWKSCni5VADnn4KzO/mbaBHTObIeOYHZxxPjt39t3Dr3xOc935n2F/0ApTJtO6sgftiv92enVEm81tnrDdDdkDjQ7wnHCoCB/nS55t5HD8DmT/QJhnnyxt2Uwr4tUMI9dxK6PIeYueNq3uSS8N+gJd/gYWKs23Cgwx9tKm08ForGVBi93yPkbD/G8k/nJbALKpc7BReCu2sBB5j30x1ijU81Ye1FGrBz5kI+Pab2h6BtjrCgGZyiSqoJ3Lsq92B5oCDKrV+TsDdLB8ntRi87Fug0CZAr6seERpsRqI4eFJLqKxBCfAofyjwLhfRCHobtPtW1BT0mVhVjL5WXCiFOiySwqoq1fmryEMKXst6sDOLtpRz2QVVl5uiqr9QLpcluEJCjjbXV1Sr5uCaJcUuGMxxxMCrZc=
```

Comment:
Is it standard to have the token like this, publicly visible ?

Reply:
yes it is encrypted : https://docs.travis-ci.com/user/encryption-keys/

Reply:
Nice